index.html

```diff
@@ -0,0 +1,236 @@
+        </noscript>
+
+        <!-- ---------- SUMMARY ---------- -->
+
+        <p>
+          <b x-data x-text="$store.store.list.length">&ndash;</b> (and counting)
+          <i>tools, packages, workflows, resources, and other software</i>
+          developed at or in collaboration with the
+          <a href="https://medschool.cuanschutz.edu/dbmi" target="_blank"
+            >Department of Biomedical Informatics at the University of
+            Colorado</a
+          >.
+        </p>
+
+        <!-- ---------- LIST ---------- -->
+
+        <div x-data class="grid" :style="{ '--cols': $store.store.cols }">
+          <template x-for="(entry, index) in $store.store.list">
+            <div
+              :style="{
+                '--offset': Math.floor(index / $store.store.cols) % 2 ? -1 : 1,
+                '--delay': (Math.random() + index / $store.store.cols / 4) + 's'
+              }"
+              class="cell"
+            >
+              <a class="hex" tabindex="0" :href="entry.link" target="_blank">
+                <!-- hexagon fill shape -->
+                <svg viewBox="-0.866 -1 1.732 2" class="shape">
+                  <path
+                    class="stroke"
+                    pathLength="1"
+                    d="
+                      M 0.866	-0.500
+                      L 0.000	-1.000
+                      L -0.866	-0.500
+                      L -0.866	0.500
+                      L 0.000	1.000
+                      L 0.866	0.500
+                      z
+                    "
+                  />
+                  <path
+                    class="fill"
+                    d="
+                      M 0.866	-0.500
+                      L 0.000	-1.000
+                      L -0.866	-0.500
+                      L -0.866	0.500
+                      L 0.000	1.000
+                      L 0.866	0.500
+                      z
+                    "
+                  />
+                </svg>
+
+                <!-- background image -->
+
+                <div class="image">
+                  <svg viewBox="-30 -22 60 40" class="img">
+                    <text
+                      x-html="entry.name.toUpperCase().split(/[^A-Z0-9]+/g).map(word => word.charAt(0)).join('')"
+                    ></text>
+                  </svg>
+                  <img
+                    class="img"
+                    :src="`images/${entry.name.toLowerCase().replace(/[^a-z0-9]+/g, '-')}.jpg`"
+                    loading="lazy"
+                    onerror="this.remove();"
+                    alt=""
+                  />
+                </div>
+
+                <!-- text content -->
+
+                <div class="text">
+                  <template x-if="entry.name">
+                    <div x-text="entry.name"></div>
+                  </template>
+
+                  <template x-if="entry.description">
+                    <div x-text="entry.description" class="truncate"></div>
```

Comment:
I was about to report that, too. I think having just the title isn't a bad idea, perhaps with a tooltip or something to display the extra text.

Reply:
Will tweak this more. Is a balance between using the space efficiently and not overflowing the hexagon, because can't make it wrap exactly to the hexagon shape (without making it significantly more complicated).